src/styles/Config.js

```diff
@@ -11,4 +11,11 @@ export const fontSizes = {
     medium: '32px',
     large: '48px',
     xLarge: '72px'
+}
+
+export const loginColors = {
```

Comment:
Why do we need another object with the same data as the 'colors' object? Can just use 'colors' instead :)